packages/server/src/api.ts

```diff
@@ -61,7 +63,17 @@ router.get("/game/:gameId/sgf", async (req, res) => {
 router.get("/games/:gameId", async (req, res) => {
   try {
     const game: GameResponse = await getGame(req.params.gameId);
-    res.send(game);
+
+    //updating player 0 rating to 700
+    const updateStatus = await updateUserRating(game.players[0].id, 700);
+    //accessing user from db to check if update was made
+    const userInfo = await getUser(game.players[0].id);
```

Comment:
Why just player 0?  Just for testing? For future reference, it will be simple to make this happen for all players:

```
const userInfos = await Promise.all(game.players.map(player => getUser(player.id));
```

But also be aware that [`game.players` its elements are nullable](https://github.com/govariantsteam/govariants/blob/d3444cb8916145464c632929aac0378ba6a8f782/packages/shared/src/api_types.ts#L16), so there should be more checking like `if (players)` and `players.filter(player => player)`

Reply:
> Just for testing?

Yes, I just wanted to make sure it worked, but I should have tested it for all players.

Reply:
Also, the rating is only updated in the DB, so whenever we pull values from the DB to display to the user, would we also have to make sure to update the `game` property with the rating because the `game` property is what is passed to the user in the API?

Reply:
> Also, the rating is only updated in the DB, so whenever we pull values from the DB to display to the user, would we also have to make sure to update the game property with the rating because the game property is what is passed to the user in the API?

I think that is something that can be decided.  But the important thing is this: if we store rating on the `game.players` object, it will not be synced to the user's current rating.  I think this is good, but just wanted to point it out.